decorators.rb

```diff
@@ -0,0 +1,26 @@
+require './nameable'
+
+class NameDecorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < NameDecorator
+  def correct_name
+    @nameable.correct_name.split(/ /).map(&:capitalize).join(' ')
+  end
+end
+
+class TrimmerDecorator < NameDecorator
+  def correct_name
+    @nameable.correct_name.strip
+  end
```

Comment:
- [ ] Kindly change the implementation of the `correct_name` method in the `TrimmerDecorator` to match this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Implement%20a%20method%20correct_name%20that%20makes%20sure%20that%20the%20output%20of%20%40nameable.correct_name%20has%20a%20maximum%20of%2010%20characters.%20If%20it%27s%20longer%20it%20should%20trim%20the%20word.). The output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word.